config/forks.csv

```diff
@@ -5,10 +5,38 @@
 ; chainname,bit,name,starttime,timeout,windowsize,threshold,minlockedblocks,minlockedtime,gbtforce
 ; main network, 95% @ 2016 blocks:
 main,0,csv,1462060800,1493596800,2016,1916,2016,0,true
-main,28,testdummy,1199145601,1230767999,2016,1916,2016,0,true
+; start sep 1, 2018, end sep 1, 2019, 90 day window, 75% threshold, 0, 90 day locked time
+main,1,block_max_size_128mb,1535760000,1567296000, 12960, 9720, 0, 7776000, true
+main,2,opcodes_mul_shift_invert,1535760000,1567296000, 12960, 9720, 0, 7776000, true
+main,3,unrestricted_script_instructions,1535760000,1567296000, 12960, 9720, 0, 7776000, true
+main,4,op_checkdatasig,1535760000,1567296000, 12960, 9720, 0, 7776000, true
+main,5,tx_min_size_100,1535760000,1567296000, 12960, 9720, 0, 7776000, true
+main,6,enforce_lexicographical_tx_order,1535760000,1567296000, 12960, 9720, 0, 7776000, true
```

Comment:
should we comment out the enforce_lexi ?  since BU members didn't vote for it; turned off by default but left in the file (commented out) so a miner could easily turn it on if they wanted to. Same for the 128 MB size?

Reply:
looks like BUIP098 will very soon pass so we should keep them in... also if you read the BUIP carefully, the vote against CTOR explicitly said not to the point where we fork away from the most-work chain.